plugins/link.md

```diff
@@ -116,9 +119,13 @@ tinymce.init({
 
 ### `link_list`
 
-This option lets you specify a predefined list of links for the link dialog. This is very useful if your users need to regularly link to the same sources.
+This option lets you specify a predefined list of links for the link dialog. These links will appear in a dropdown menu in the link dialog, and when clicked will auto-populate the dialog with the relevant data. This is useful if your users need to regularly link to the same sources.
```

Comment:
> These links will appear in a dropdown menu in the link dialog, and when clicked will auto-populate the dialog with the relevant data. 

Maybe:
These links are added to a drop-down list in the link dialog. When a list item is selected, the dialog will be populated with the relevant data.